js/components/bigButtonMaker.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Creates a button of a specific type.
+ * @param {object} props - The properties of the big button.
+ * @param {string} props.type - The type of button ("proceed", "back").
+ * @param {string} props.text - The text to be displayed on the button.
+ * @param {void} props.onClick - Function to be executed on clicking the button.
+ * @returns {HTMLElement} - The generated button element.
+ * @throws {Error} - Throws an error if the type is not "proceed" or "back".
+ */
+
+export function bigButtonMaker(props) {
+   const type = props.type;
+   const text = props.text;
+
+   if (type !== 'proceed' && type !== 'back') {
```

Comment:
**nitpick:** If you differ background color by passing string, why don't you pass just background color, as a property? Currently when you would have to add new colors, you'd have to add new string variants - and add more ifs. It would be much more simple to to like this, and remove this `if`. You don't need it

```
const type = props.type;
const text = props.text;
const bgColor = props.bgColor;
```